src/components/Tab/Tabs.js

```diff
@@ -0,0 +1,35 @@
+import React from "react";
+import { useState } from "react";
+import StyledTabs from "./Tabs.styled";
+import tabsSettings from "./tabsSettings";
+import Tab from "./Tab";
+
+const Tabs = () => {
+  const [isActive, setActive] = useState(false);
+  const clickHandler = (id) => {
+    console.log(id);
+    isActive ? setActive(false) : setActive(true);
+  };
```

Comment:
Twój `state` przechowuje tylko informacje czy `true/false` czyli ma tylko 2 możliwe wartości. Natomiast "tabów" jest więcej.
Zwróć uwagę, że w Boostrapie masz coś takiego jak `eventKey="home"` przypisany do każdej zakładki (`tab`).
To właśnie jest identyfikator każdej za zakładki.
To ten element powinieneś zapisywać w state i na jego podstawie określasz co jest widoczne np.

```
const [active, setActive] = useState(defaultActiveKey);
// ...
<Tab active={active === eventKey} ... />
```
PS. Pewnie będzie trzeba przebudować to rozwiązanie na `children` i wykorzystać te  rozwiązania: https://www.smashingmagazine.com/2021/08/react-children-iteration-methods/